test/blocks/library-config/mocks/blocks/container/body-mixed.html

```diff
@@ -0,0 +1,208 @@
+                <h2 id="melon">Melon</h2>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</p>
+                <p><strong><a href="https://www.adobe.com/">Learn about</a></strong></p>
+            </div>
+        </div>
+    </div>
+    <div class="section-metadata">
+        <div>
+            <div>carousel</div>
+            <div>Carousel container</div>
+        </div>
+        <div>
+            <div>style</div>
+            <div>dark</div>
+        </div>
+        <div>
+            <div>background</div>
+            <div>
+                <picture>
+                    <source type="image/webp" srcset="" media="(min-width: 600px)">
+                    <source type="image/webp" srcset="">
+                    <source type="image/png" srcset="" media="(min-width: 600px)">
+                    <img loading="lazy" alt="" type="image/png" src="" width="950" height="700">
+                </picture>
+            </div>
+        </div>
+    </div>
+</div>
+<div>
+    <div class="text large xl-spacing">
+        <div>
+            <div>
+                <h2 id="avocado">Avocado</h2>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</p>
+                <p><strong><a href="https://www.adobe.com/">Learn about</a></strong></p>
+            </div>
+        </div>
+    </div>
+    <div class="section-metadata">
+        <div>
+            <div>carousel</div>
+            <div>Carousel container</div>
+        </div>
+        <div>
+            <div>style</div>
+            <div></div>
+        </div>
+        <div>
+            <div data-align="right">background</div>
+            <div>
+                <picture>
+                    <source type="image/webp" srcset="" media="(min-width: 600px)">
+                    <source type="image/webp" srcset="">
+                    <source type="image/png" srcset="" media="(min-width: 600px)">
+                    <img loading="lazy" alt="" type="image/png" src="" width="950" height="700">
+                </picture>
+            </div>
+        </div>
+    </div>
+    <div class="library-container-end">
+        <div>
+            <div></div>
+        </div>
+    </div>
+</div>
+<div>
+    <h2 id="carousel-lightbox">Carousel (lightbox)</h2>
+    <p>Full width carousel with lightbox feature</p>
+    <div class="library-container-start">
+        <div>
+            <div></div>
+        </div>
+    </div>
+    <div class="carousel lightbox">
+        <div>
+            <div>Carousel lightbox</div>
+        </div>
+    </div>
+    <div class="section-metadata">
+        <div>
+            <div>style</div>
+            <div>xxl spacing</div>
+        </div>
+    </div>
+</div>
+<div>
+    <h3 id="avocado-surprise">Avocado surprise</h3>
+    <p>
+        <picture>
+            <source type="image/webp" srcset="./media_184725b8b0d0accc1cdf782812e7815d356197668.png?width=2000&amp;format=webply&amp;optimize=medium" media="(min-width: 600px)">
```

Comment:
Please do not use actual links to uploaded media, these can lead to 404 and cause unit-test issues. Instead, either use an existing mock image or an empty `srcset` in case you do not rely on the presence of an image for the test.

cc: @mokimo

Reply:
Yup, very good point see https://github.com/orgs/adobecom/discussions/814#discussioncomment-6061072

Reply:
Addressed in https://github.com/adobecom/milo/pull/772/commits/e59df2f53ee5fade316b26cabb26adfb6191362a